src/web/routes/application/postcode/postcode.js

```diff
@@ -8,6 +8,25 @@ const REQUEST_TIMEOUT = 5000
 
 const standardisePostcode = (postcode) => postcode.toUpperCase().replace(/\s/g, '')
 
+const buildGoogleAnalyticsFullUri = (gaBaseUrl, trackingId, result, ipAddress, numberOfResults, sessionId) =>
+  `${gaBaseUrl}?v=1&tid=${trackingId}&t=event&cid=${sessionId}&ec=AddressLookup&ea=${result}&el=${ipAddress}&ev=${numberOfResults}`
+
+const auditSuccessfulPostcodeLookup = async (config, req, numberOfResults) => auditPostcodeLookup(config, req, 'SuccessfulLookup', numberOfResults)
+
+const auditFailedPostcodeLookup = async (config, req) => auditPostcodeLookup(config, req, 'FailedLookup', 0)
+
+const auditPostcodeLookup = async (config, req, result, numberOfResults) => {
```

Comment:
Maybe 'result' could be renamed 'outcome' to make it different to 'numberOfResults'

Reply:
Done